src/Impostor.Server/Net/Inner/Objects/InnerMeetingHud.cs

```diff
@@ -80,7 +81,15 @@ public override async ValueTask HandleRpc(ClientPlayer sender, ClientPlayer? tar
                     var tie = reader.ReadBoolean();
 
                     await _eventManager.CallAsync(new MeetingEndedEvent(_game, this));
-
+                    if(playerId != 0xFF){ // make sure a player was voted out
+                        var player = _game.GameNet.GameData.GetPlayerById(playerId);
+                        if(player != null) // fixes a warning
+                        {
+                            player.IsDead = true; // mark player as dead
+                            player.LastDeathReason = Api.Innersloth.DeathReason.Exile; // set death reason
+                            await _eventManager.CallAsync(new PlayerExileEvent(_game,sender,player.Controller)); // call PlayerExileEvent
```

Comment:
No need for the comments after each of these lines, the code is self explanatory.